.github/workflows/pull_request.yml

```diff
@@ -268,7 +268,7 @@ jobs:
         run: COMPOSER_PREFER_LOWEST=${{ matrix.prefer-lowest }} composer install --prefer-dist --no-interaction --no-progress
 
       - name: Execute tests
-        run: vendor/bin/phpunit --coverage-text --show-uncovered-for-coverage-text
+        run: vendor/bin/phpunit --order-by=random --coverage-text --show-uncovered-for-coverage-text
```

Comment:
We had some inter-dependent tests. This should help expose these sooner.

I generally like a deterministic testsuite, however I feel this is a good change to expose unintentional global state dependencies between tests.